### DISCUSSION TITLE
I can not reverify my academic affiliation

### DISCUSSION THREAD

**Discussion by slantingsun** [community/community#78108] [GitHub Education]
### Hi everyone,

when I click on the  https://education.github.com/discount_requests/application link，I will be redirected to the https://github.com/pricing link

**Comment by mdmuhtasimfuadfahim** [comment]
@ETOPDC Please add your university email first and make it primary. Also change your GitHub profile name and give it same as your university id name. Then try to visit [here](https://education.github.com/discount_requests/application) and you need to upload a image of you with your student id card.

**Comment by Lyuansama** [comment]
I met the same question. Did you sovle your problem with the above method? @ETOPDC

**Comment by Penguin-SAMA** [comment]
the same. Did you sovle? @ETOPDC

**Comment by anshumancodes** [comment]
Am facing the same issue since 1 month now

**Comment by Akasaz** [comment]
same problem, and I still can't reverify my academic affiliation at present.

**Comment by mdmuhtasimfuadfahim** [comment]
If you need a urgent fix please open a ticket [here](https://support.github.com/) or [email](https://github.com/orgs/community/discussions/support@github.com) them directly. I guess this will drive into a quick solution.

**Comment by xsm1997** [comment]
I opened a support ticket about this, but it is closed silently by Github after some days without any comment. I replied and opened the ticket again, asking for reason, but it is closed by Github again after a day, also without any comment.

**Comment by Penguin-SAMA** [comment]
> If you need a urgent fix please open a ticket [here](https://support.github.com/) or [email](https://github.com/orgs/community/discussions/support@github.com) them directly. I guess this will drive into a quick solution.

I have created a ticket, but many days have passed and still no one has replied to me.
